src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -93,6 +93,9 @@ export class RbdFormComponent extends CdForm implements OnInit {
     '16 MiB',
     '32 MiB'
   ];
+
+  defaultStripingCount = 1;
```

Comment:
Good one!: using constants instead of hardcoding literals across the code ;-)

Reply:
Thanks , I have updated the code to do the same with defaultStripingUnit